Tools/clinic/clinic.py

```diff
@@ -2041,7 +2044,7 @@ def __repr__(self) -> str:
 
     def clear(self) -> None:
         if self.type != 'buffer':
-            fail("Can't clear destination" + self.name + " , it's not of type buffer")
+            fail(f"Can't clear destination {self.name!r}, it's not of type buffer")
```

Comment:
This would be better grammar:

```suggestion
            fail(f"Can't clear destination {self.name!r}: it's not of type buffer")
```